setups/WindowBuilder.setup

```diff
@@ -61,6 +61,14 @@
         <value>record</value>
       </detail>
     </annotation>
+    <setupTask
+        xsi:type="setup:CompoundTask"
+        name="org.eclipse.jdt.ui">
+      <setupTask
+          xsi:type="setup:PreferenceTask"
+          key="/instance/org.eclipse.jdt.ui/spelling_user_dictionary"
+          value="${workspace.location|uri}/git/windowbuilder/org.eclipse.wb.core/dictionary.txt"/>
```

Comment:
I don’t think you should assume a relationship between the workspace location and the clone location.  Is the value supposed to be a uri?

Reply:
Use the clone location 

https://github.com/eclipse-windowbuilder/windowbuilder/blob/0f89c110d82e9b5bea7368b71341ef97177c1762/setups/WindowBuilder.setup#L142

You can use |uri with that if the path should be converted to a uri.

Reply:
Using the clone location makes it look a lot cleaner. Thanks!